scripts/onboarding/aks/syslogonboarding/existingClusterOnboarding.json

```diff
@@ -0,0 +1,274 @@
+                                        "Microsoft-ContainerInsights-Group-Default"
+                                    ],
+                                    "destinations": [
+                                        "ciworkspace"
+                                    ]
+                                }
+                            ]
+                        }
+                    }
+                ]
+              },
+        "syslogAndCITemplate": {
+                "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                "contentVersion": "1.0.0.0",
+                "parameters": {},
+                "variables": {},
+                "resources": [
+                    {
+                        "type": "Microsoft.Insights/dataCollectionRules",
+                        "apiVersion": "2021-04-01",
+                        "name": "[variables('dcrName')]",
+                        "location": "[parameters('workspaceLocation')]",
+                        "tags": "[parameters('resourceTagValues')]",
+                        "kind": "Linux",
+                        "properties": {
+                            "dataSources": {
+                                "syslog": [
+                                    {
+                                        "streams": [
+                                            "Microsoft-Syslog"
+                                        ],
+                                        "facilityNames": [
+                                            "auth",
+                                            "authpriv",
+                                            "cron",
+                                            "daemon",
+                                            "mark",
+                                            "kern",
+                                            "local0",
+                                            "local1",
+                                            "local2",
+                                            "local3",
+                                            "local4",
+                                            "local5",
+                                            "local6",
+                                            "local7",
+                                            "lpr",
+                                            "mail",
+                                            "news",
+                                            "syslog",
+                                            "user",
+                                            "uucp"
+                                        ],
+                                        "logLevels": [
+                                            "Debug",
+                                            "Info",
+                                            "Notice",
+                                            "Warning",
+                                            "Error",
```

Comment:
isnt that single log level here? why do we need different log levels here?

Reply:
why single log level? we are covering all types of log levels and this is taken from the DCR that is currently being created by AMA for VMs